### DISCUSSION TITLE
iBMi Notebooks

### DISCUSSION THREAD

**Discussion by JonasEngstedt** [codefori/vscode-ibmi#1642] [General]
Hi!

Just managed to convince my company to let me start using Code4i instead of the RDi, still a long process to go to be able to connect everything but I've come long way. I recently saw Liam's talks during Common Sweden and got real interested in iBMi Notebooks. But after searching far and wide I can't seem to find any extension for it as shown in the youtube tutorials and can't seem to get vscode to recognize the inb extension. Is this discontinued or not available in Sweden or something? Cheers.

**Comment by worksofliam** [comment]
@JonasEngstedt Your timing is amazing.

Notebooks are currently being moved into the database extension. The current state of IBM i Notebooks was not great and I want to make sure the experience is as good as possible.

If you have the database extension, keep an eye out for future updates for it to be available.

Sorry for the trouble!
